src/ceph-daemon

```diff
@@ -726,7 +728,7 @@ class CephContainer:
 
 def command_version():
     out = CephContainer(args.image, 'ceph', ['--version']).run()
-    print(out, end='')
+    print(out),
```

Comment:
there is a trailing comma here, and this will not work in Python2 unless this line is added at the very top:

```python
from __future__ import print_function
```

Reply:
it seems to run without that, but hte behavior is not consistent:
```
gnit:build (wip-osdmon-iec) 12:11 PM $ cat /tmp/foo.py 
print('a'),
print('b')
gnit:build (wip-osdmon-iec) 12:11 PM $ python2 /tmp/foo.py 
a b
gnit:build (wip-osdmon-iec) 12:11 PM $ python3 /tmp/foo.py 
a
b
```
if i add that line,
```
gnit:build (wip-osdmon-iec) 12:12 PM $ cat /tmp/foo.py 
from __future__ import print_function
print('a'),
print('b')
gnit:build (wip-osdmon-iec) 12:12 PM $ python2 /tmp/foo.py 
a
b
gnit:build (wip-osdmon-iec) 12:12 PM $ python3 /tmp/foo.py 
a
b
```
but i think i can avoid this whole mess if i just strip the newline off the string and use normal print.  i'll do that!

Reply:
updated